src/libraries/System.Security.Cryptography.Xml/src/System/Security/Cryptography/Xml/CanonicalXmlDocument.cs

```diff
@@ -61,52 +61,54 @@ public void WriteHash(HashAlgorithm hash, DocPosition docPos, AncestralNamespace
             }
         }
 
+#pragma warning disable CS8765 // Nullability doesn't match overriden member
         public override XmlElement CreateElement(string prefix, string localName, string namespaceURI)
+#pragma warning restore CS8765
         {
-            return new CanonicalXmlElement(prefix, localName, namespaceURI, this, _defaultNodeSetInclusionState);
+            return new CanonicalXmlElement(prefix!, localName, namespaceURI!, this, _defaultNodeSetInclusionState);
```

Comment:
Note my previous comments about annotating constructors on these Canonical types and avoiding these `!`s apply to all of these cases, too, not just CanonicalXmlCDataSection.

Reply:
Yes, you're right. I spotted those shortly after committing. I've checked the others and I _think_ there's no more.